### DISCUSSION TITLE
tfenv não executa sem formato

### DISCUSSION THREAD

**Discussion by viniciusczar** [chgasparoto/curso-aws-com-terraform#63] [Q&A]
Professor,
 Ao instalar o tfenv no windows e colocar do jeito que está indiciado, a pasta bin nas variáveis, executando qualquer comando é perguntado em lista (do windows) com qual arquivo executar o programa, logo, não é reconhecido o formato.

**Comment by chgasparoto** [comment]
@viniciusczar acabei de responder você no Youtube.

> Aqui tem algumas alternativas para tentar fazer funcionar Vinicius. Eu não testei para ver se funciona pois estou sem uma máquina Windows.
> 
> https://dev.to/lkurzyniec/terraform-version-switcher-for-windows-42c4
> https://github.com/warrensbox/terraform-switcher
